multiview/embed/mvmds.py

```diff
@@ -0,0 +1,217 @@
+# Copyright 2019 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from multiview.embed.base import BaseEmbed
+from multiview.utils.utils import check_Xs
+import warnings
+import numpy as np
+from sklearn.metrics import euclidean_distances
+
+
+class MVMDS(BaseEmbed):
+
+    """"
+    Classical Multiview Multidimensional Scaling for jointly reducing
+    the dimensions of multiple views of data. A euclidean distance matrix
+    is created for each view, double centered, and the k largest common
+    eigenvectors are returned based on the algorithm proposed by the
+    following paper:
+
+    https://www.sciencedirect.com/science/article/pii/S016794731000112X
+
```

Comment:
In this top level comment, specify parameters and attributes used. For example, n_components should go here and number of iterations.